docs/source/indexes/cagra.rst

```diff
@@ -98,53 +98,143 @@ Tuning Considerations
 The 3 hyper-parameters that are most often tuned are `graph_degree`, `intermediate_graph_degree`, and `itopk_size`.
 
 Memory footprint
-----------------
+================
 
-CAGRA builds a graph that ultimately ends up on the host while it needs to keep the original dataset around (can be on host or device).
+CAGRA builds a nearest-neighbor graph(neighbor IDs) that ultimately ends up on the host while it needs to keep the original dataset(raw vectors) around (can be on host or device)
 
-IVFPQ or NN-DESCENT can be used to build the graph (additions to the peak memory usage calculated as in the respective build algo above).
+Baseline Memory Footprint
+--------------
 
-Dataset on device (graph on host):
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. math::
```

Comment:
Before we merge this- just want to veirfy that you did build the docs locally and checked that the math looks as expected?